Cargo.toml

```diff
@@ -11,7 +11,7 @@ homepage = "https://qdrant.tech/"
 repository = "https://github.com/qdrant/qdrant"
 license = "Apache-2.0"
 edition = "2021"
-rust-version = "1.77"
+rust-version = "1.80"
```

Comment:
Required for [`div_duration_f32`] that I now use in consensus loop.

[`div_duration_f32`]: https://doc.rust-lang.org/std/time/struct.Duration.html#method.div_duration_f32